test/dlc_tests/sanity/test_pre_release.py

```diff
@@ -723,10 +723,10 @@ def test_pip_check(image):
         framework_version
     ) in SpecifierSet(">=2.9.1"):
         exception_strings = []
-        models_versions = ["2.9.1", "2.9.2", "2.10.0", "2.11.0", "2.12.0", "2.13.0"]
+        models_versions = ["2.9.1", "2.9.2", "2.10.0", "2.11.0", "2.12.0", "2.13.0", "2.14.2"]
         for ex_ver in models_versions:
             exception_strings += [f"tf-models-official {ex_ver}".replace(".", "\.")]
-        text_versions = ["2.9.0", "2.10.0", "2.11.0", "2.12.0", "2.13.0"]
+        text_versions = ["2.9.0", "2.10.0", "2.11.0", "2.12.0", "2.13.0", "2.14.0"]
```

Comment:
The current merge conflict in PR is caused by these two lists being in-lined into their corresponding for-loops instead of being kept as a separate list variable. Will resolve the merge conflict in the next revision, but it does not affect the functionality of the code in any way.